mtp_api/apps/transaction/utils.py

```diff
@@ -37,9 +37,9 @@ def format_number(value, truncate_after=None):
     """
     if truncate_after is not None and value > truncate_after:
         if value >= 1000000:
-            return '{:,}m'.format(value / 1000000)
+            return '{:0,.1}m'.format(value / 1000000)
         if value >= 1000:
-            return '{:,}k'.format(value / 1000)
+            return '{:0,.1}k'.format(value / 1000)
```

Comment:
I'm always finding these format strings a little bit confusing/terse.

I'm guessing the intent of that `,.1` here is to round to the 1st decimal place.
I've tried in the console and it spits out scientific notation (which would confuse most people 😬), e.g.:

```python
>>> '{:,.1}k'.format(100.244089)
'1e+02k'
```

But it rounds to the 1st decimal when adding an `f` instead, e.g.:

```python
>>> '{:,.1f}k'.format(100.244089)
'100.2k'
```

Reply:
you're quite right! i copied it wrong from the currency thing, but also wasn't aware that it would use scientific notation otherwise